modules/p2/ding_list/ding_list.install

```diff
@@ -37,6 +37,12 @@ function ding_list_install() {
   $mail_settings = variable_get('mail_system', array());
   $mail_settings['ding_list_share'] = "HTMLMailSystem";
   variable_set('mail_system', $mail_settings);
+
+  module_load_include('inc', 'page_manager', 'plugins/tasks/page');
+  $page = page_manager_page_load('user_bookmarks');
+  if (!empty($page)) {
+    page_manager_page_delete($page);
+  }
```

Comment:
Why do we need to delete the page on install?

As the configuration is deleted from the feature export I cannot see how there would be anything to delete when new sites are installed.